db/db.go

```diff
@@ -1050,3 +1056,81 @@ func (db database) GetUserAssignedOrganizations(pubkey string) []OrganizationUse
 	db.db.Where("owner_pub_key = ?", pubkey).Find(&ms)
 	return ms
 }
+
+func (db database) AddBudgetHistory(budget BudgetHistory) BudgetHistory {
+	db.db.Create(&budget)
+	return budget
+}
+
+func (db database) CreateOrganizationBudget(budget BountyBudget) BountyBudget {
+	db.db.Create(&budget)
+	return budget
+}
+
+func (db database) UpdateOrganizationBudget(budget BountyBudget) BountyBudget {
+	db.db.Where("organization = ?", budget.Organization).Updates(budget)
```

Comment:
same thing here, what if two organizations have the same name

Reply:
oh!, The organization column is for the organizationUUID not the organization name, and there is also a check during organization creation for existing organization names